internal/integration/actions.go

```diff
@@ -224,18 +226,20 @@ func ExpectError(action Action, expectedErrorMsg ...string) Action {
 }
 
 // Deploy a module from the working directory and wait for it to become available.
-func Deploy(module string) Action {
+func Deploy(modules ...string) Action {
 	return Chain(
 		func(t testing.TB, ic TestContext) {
 			args := []string{"deploy", "-t", "4m"}
 			if ic.kubeClient.Ok() {
 				args = append(args, "--build-env", "GOOS=linux", "--build-env", "GOARCH=amd64", "--build-env", "CGO_ENABLED=0")
 			}
-			args = append(args, module)
+			args = append(args, modules...)
 
 			Exec("ftl", args...)(t, ic)
 		},
-		WaitWithTimeout(module, time.Minute*2),
+		Chain(islices.Map(modules, func(module string) Action {
```

Comment:
How is his parallel though?

Reply:
That part is still serial, but actually yeh I make that parallel as well.